include/fastrtps/xmlparser/XMLProfileManager.h

```diff
@@ -230,27 +230,31 @@ class XMLProfileManager
 
 private:
 
-    RTPS_DllAPI static XMLP_ret extractDynamicTypes(
+    static XMLP_ret loadXMLFile(
+            const std::string& filename,
+            const bool& is_default);
```

Comment:
Why are you using `const bool&` ?

```suggestion
            bool is_default);
```

Reply:
You are right. My mistake.